content/en/docs/collector/getting-started.md

```diff
@@ -1,358 +1,88 @@
-`{{% param collectorVersion %}}` with the version of the Collector you wish to
-run.
+1. Download and run the OpenTelemetry Collector Docker container:
 
-{{< tabpane text=true >}} {{% tab DockerHub %}}
+   ```sh
+   docker pull otel/opentelemetry-collector:{{% param collectorVersion %}}
+   docker run -p 127.0.0.1:4317:4317 -p 127.0.0.1:55679:55679 otel/opentelemetry-collector:{{% param collectorVersion %}}
+   ```
 
-```sh
-docker pull otel/opentelemetry-collector-contrib:{{% param collectorVersion %}}
-docker run otel/opentelemetry-collector-contrib:{{% param collectorVersion %}}
-```
+2. Download and install the telemetrygen utility from the
+   [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/)
+   repository:
 
-{{% /tab %}} {{% tab ghcr.io %}}
+   ```sh
+   go install github.com/open-telemetry/opentelemetry-collector-contrib/cmd/telemetrygen@latest
+   ```
 
-```sh
-docker pull ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:{{% param collectorVersion %}}
-docker run ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:{{% param collectorVersion %}}
-```
+3. The telemetrygen command generates dummy telemetry for testing. Try sending
+   some traces to the Collector:
 
-{{% /tab %}} {{< /tabpane >}}
+   ```sh
+   telemetrygen traces --otlp-insecure --duration 5s
+   ```
 
-To load your custom configuration `config.yaml` from your current working
-directory, mount that file as a volume:
+   After five seconds, `telemetrygen` stops and shows the sended messages in the
+   console:
 
-{{< tabpane text=true >}} {{% tab DockerHub %}}
+   ```text
+   2023-10-23T12:58:19.835+0200	INFO	traces/worker.go:88	traces generated	{"worker": 0, "traces": 994418}
+   2023-10-23T12:58:19.835+0200	INFO	traces/traces.go:79	stop the batch span processor
+   ```
 
-```sh
-docker run -v $(pwd)/config.yaml:/etc/otelcol-contrib/config.yaml otel/opentelemetry-collector-contrib:{{% param collectorVersion %}}
-```
+   In the terminal window, running the Collector container, logs should show
+   activity related to the trace ingest similar to this:
 
-{{% /tab %}} {{% tab ghcr.io %}}
+   ```text
+   Span #434
+      Trace ID       : ba7ef95fce7499811ca72158350c907c
+      Parent ID      : 1d3c9f49b3f2cf47
+      ID             : 7609079dc6253034
+      Name           : okey-dokey
+      Kind           : Server
+      Start time     : 2023-10-23 11:01:29.53251 +0000 UTC
+      End time       : 2023-10-23 11:01:29.532634 +0000 UTC
+      Status code    : Unset
+      Status message :
+   ```
 
-```sh
-docker run -v $(pwd)/config.yaml:/etc/otelcol-contrib/config.yaml ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:{{% param collectorVersion %}}
-```
+4. Open `http://localhost:55679/debug/tracez` in your browser and select one of
+   the samples in the table to see the traces you've just generated.
```

Comment:
@theletterf just a suggestion (since we have to wait ;-)): what about adding optional extra steps for metrics and logs as well (if telemetrygen can do them)

Reply:
Good point. Would tabs be adequate in this case?

Reply:
Whoops. Status of metrics and logs is still "development". I wouldn't try adding them to the tutorial. +CC @mx-psi

Reply:
Then let's not add them :-)